src/Classes/Utils/ColorUtils.ts

```diff
@@ -0,0 +1,199 @@
+import { IColor } from "../../Interfaces/Options/Particles/IColor";
+import { IRgb } from "../../Interfaces/IRgb";
+import { IRgba } from "../../Interfaces/IRgba";
+import { IHsl } from "../../Interfaces/IHsl";
+import { IHsla } from "../../Interfaces/IHsla";
+import { Utils } from "./Utils";
+
+export class ColorUtils {
+    /**
+     * Gets the particles color
+     * @param color the input color to convert in [[IRgb]] object
+     */
+    public static colorToRgb(color: IColor): IRgb | undefined {
+        let res: IRgb | undefined;
+
+        if (typeof (color.value) === "string") {
+            if (color.value === "random") {
+                res = {
+                    b: Math.floor(Math.random() * 256),
+                    g: Math.floor(Math.random() * 256),
+                    r: Math.floor(Math.random() * 256),
+                };
+            } else {
+                res = ColorUtils.stringToRgb(color.value);
+            }
+        } else {
+            if (color.value instanceof Array) {
+                const arr = color.value as string[];
+                const colorSelected = color.value[Math.floor(Math.random() * arr.length)];
+
+                res = ColorUtils.stringToRgb(colorSelected);
+            } else {
+                const rgbColor = color.value as IRgb;
+
+                if (rgbColor.r !== undefined) {
+                    res = rgbColor;
+                }
+
+                const hslColor = color.value as IHsl;
+
+                if (hslColor.h !== undefined) {
+                    res = ColorUtils.hslToRgb(hslColor);
+                }
+            }
+        }
+
+        return res;
+    }
+
+    public static stringToAlpha(input: string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'stringToAlpha' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)